app/src/main/java/com/team3663/scouting_app/activities/PreMatch.java

```diff
@@ -323,6 +324,9 @@ private void initNext() {
                     Globals.EventLogger.LogData(Constants.Logger.LOGKEY_SCOUTER, preMatchBinding.editScouterName.getText().toString().toUpperCase().replace(" ",""));
                     Globals.EventLogger.LogData(Constants.Logger.LOGKEY_DID_PLAY, String.valueOf(preMatchBinding.checkboxDidPlay.isChecked()));
                     Globals.EventLogger.LogData(Constants.Logger.LOGKEY_TEAM_SCOUTING, String.valueOf(Globals.CurrentScoutingTeam));
+
+                    Achievements.data_TeamToScout = Globals.CurrentTeamToScout;
```

Comment:
Why are you doing this inside of the Init_Next and not the Init_Achievments?

Reply:
There is no Inii_Achievement in pre-Match activity.  This is where we set the Global value, so I stored it in the Achievement class right away so we don't have to worry about the timing of when it was set.

Reply:
There is a Init_Acheivments in pre-Match... Its line 420.

Reply:
oh haha..   I was looking at the code in my other branch, and it didn't have it.  oops.  I could move it.  The general idea throughout the code was to update the achievement data where the "thing" happened.  In this case, there happens to be a function we call right after, so it COULD go there, but all of the other code sets the achievement data as soon as it can.